nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
I feel like there should be argument checking of some sort in the classes here, and at least give some sort of warning. It seems very easy to unknowingly pass in parameters that are completely unused.

The `get()`s make me worry that someone may pass in a typoed argument like "nin_duration" (instead of "min_duration") that gets silently discarded and set to a default.

(Unless omegaconf checks this somehow with a Config class...?)

Reply:
You're right, theres no check at the moment (and without structured config for these corresponding classes, Omegaconf will silently pass them along and the get will use the default). I think we need to add dataclasses to these datasets asap (and the other ones too).